### DISCUSSION TITLE
Добавить в начальную стадию запуска Сypress

### DISCUSSION THREAD

**Discussion by Vivioza5** [open-tutorials/cypress#31] [Ideas]
Возможно сильно преувеличиваю, но из чувства эмпатии предлагаю в первоначальные действия которые надо сделать перед работой над почти всеми учебниками стоит добавить команду запуска cypress

- Выполни команду `npx cypress open` 

Потому как и на стендапе слышал и сам сталкивался с этим, тесты создаются и складываются в папку `%/projects/cypress/integration`  а без первого запуска  она не будет создана и как начинающим кодерам надо вспомнить, что это именно папка integration и она будет после запуска cypress.

С другой стороны сам прошел пару раз через эту боль)) быстрее запомнилось, что надо сделать первый запуск)) если идея не пригодится, то в дискуссии она может будет висеть для напоминания об этом действии))

**Comment by Vladimir-Tarasov** [comment]
<img width="600" alt="Screenshot 2023-02-02 at 16 36 16" src="https://user-images.githubusercontent.com/92365254/216369653-81708a71-87d3-4627-8016-b7fc90917b36.png">   

Ссылка на подробное описание установки и первый запуск Cypress  есть в каждом туториале.

**Comment by breslavsky** [comment]
Я думаю без боли не будет знаний :-)
